### ISSUE TITLE
Finetune on ImageNet3D dataset

### ISSUE THREAD

**Issue by jcy-54** [ByteDance-Seed/Depth-Anything-3#275]
Hi, 

Thanks for the great work! Recently I noticed there is a ImageNet3D dataset (https://github.com/Rawmantic/Dataset3D), which annoted with pose, normal and depth. I'm able to improve foundationstereo greatly with such dataset, but for depthanythingv3, I'm wondering how to finetune it with improved foundationstereo label, or use that dataset directly. Is there any training script and data preparation I can refer to?

Thanks!
